### PR TITLE
Ignition 7.9 54 update ignition 7.9 readme

### PR DESCRIPTION
**Pull request by alexjhawk**

### CODE REVIEW

**Comment by alexjhawk** [comment]
Waiting for the other two PRs to get merged before marking this ready. The images need to be replaced with 7.9 equivalents, but the other PRs change what the contents of the images would be.

File: readme.md [hunk 1]
```diff
@@ -1,46 +1,257 @@
...
+
+### **Realtime Values**
+
+The realtime value functionality allows Ignition to read live tag values from an Ewon up to once per second.
+
+This functionality can be enabled in three ways.
+
+**1. Individual Tag**
+
+   * Open Designer and navigate to the `Tag Browser`
```
Comment (TomKimsey) [inline readme.md]: Not possible in 7.9

**Comment by alexjhawk** [comment]
> Images are from Ignition 8 not 7.9

Changed now. That's why I'd left this in draft until the other two PRs were merged.

**Comment by TomKimsey** [comment]
> > Images are from Ignition 8 not 7.9
> 
> Changed now. That's why I'd left this in draft until the other two PRs were merged.

oh okay, I did not see that it was a draft, only that my review was requested.  Is it still a draft or should I review?

**Comment by alexjhawk** [comment]
> > > Images are from Ignition 8 not 7.9
> > 
> > 
> > Changed now. That's why I'd left this in draft until the other two PRs were merged.
> 
> oh okay, I did not see that it was a draft, only that my review was requested. Is it still a draft or should I review?

This is set to review now.

**Comment by TomKimsey** [comment]
auth_info has a mouse cursor in it

**Comment by TomKimsey** [comment]
auth_error has the cursor, not auth_info

**Comment by alexjhawk** [comment]
> auth_error has the cursor, not auth_info. auth_info doesnt even exist.

fixed.

**Pull request merged**